docs/documentation/server_admin/topics/oid4vci/vc-issuer-configuration.adoc

```diff
@@ -0,0 +1,361 @@
+== Configuring Keycloak as a Verifiable Credential Issuer
+
+{project_name} provides support for https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html[OpenID for Verifiable Credential Issuance].
+
+=== Introduction
+
+This chapter provides step-by-step instructions for configuring Keycloak as a Verifiable Credential Issuer using the OpenID for Verifiable Credential Issuance (OID4VCI) protocol. It outlines the process for setting up a Keycloak instance to securely issue and manage Verifiable Credentials (VCs), supporting decentralized identity solutions.
```

Comment:
Is it possible to replace `Keycloak` with `{project_name}` on all places of this PR? See other documents from the Keycloak documentation for the inspiration.